skyportal/models.py

```diff
@@ -3448,6 +3454,346 @@ class SourceNotification(Base):
     level = sa.Column(sa.String(), nullable=False)
 
 
+class GcnNotice(Base):
+    """Records of ingested GCN notices"""
+
+    create = read = public
+    update = delete = AccessibleIfUserMatches('sent_by')
+
+    id = sa.Column(sa.Integer, primary_key=True, autoincrement=True)
```

Comment:
```suggestion
```
If you use `DBSession().add()` rather than `merge()` for adding new records, you shouldn't need to redefine this here

Reply:
Ah yeah, that could explain the errors you were seeing with getting rid of the `id` column